crates/storage/store.rs

```diff
@@ -43,13 +42,25 @@ pub enum EngineType {
     RedB,
 }
 
-impl Store {
-    pub async fn store_changes(&self, query_plan: QueryPlan) -> Result<(), StoreError> {
-        self.engine.store_changes(query_plan).await
+type StorageNode = (NodeHash, Vec<u8>);
+
+pub struct DBUpdateBatch {
+    pub account_updates: Vec<(NodeHash, Vec<u8>)>,
+    pub storage_updates: Vec<(H256, Vec<StorageNode>)>,
```

Comment:
Maybe rename `StorageNode` to `TrieNode`, as that name is generic enough to use for both `account_updates` and `storage_updates`-